### ISSUE TITLE
Navigate to metric tabs directly

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#4172]
# User story

As experienced user I want to open a specific metric tab by clicking on a measurement column in the subject table so that I don't have to navigate to the tab myself

## Blocked by

- #8825
- #8826

## Out of scope

- Visual cues for this functionality (it's aimed at experienced users that have either discovered this feature themselves or read about it in the user manual).

# Tasks

### Frontend

- [x] Add click handlers to the relevant table cells.

## Documentation

- [x] Update changelog

Reported by @Sebastiaan127001

**Comment by Sebastiaan127001** [comment]
%VOTE

**Comment by Sebastiaan127001** [comment]
@fniessink clicking on comments directs user to comment field

**Comment by Sebastiaan127001** [comment]
@fniessink if we choose to change the order of the tabs and to have the measurement tab first, we can use this feature: https://react.semantic-ui.com/elements/placeholder/
